text/0000-alternate-registries.md

```diff
@@ -102,6 +107,12 @@ plus has the ability to have crates pushed to it, however this has the following
 * It requires crates.io to be able to combine two registries, or requires a radical change to the way crates.io works
 * The current proposal could be extended to support this, if a caching server is added at a later stage
 
+## Including registry definitions in a global location
+We considered using a global configuration file (eg ~/.cargo/config) to allow a registry to
```

Comment:
I don't think broad "we believe" statements like this are as truthful as they could be.

Future readers will no doubt read this as something everyone in this discussion agreed on which isn't the case.